lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,17 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, from: nil, till: nil, outcome: nil, key: nil, primary: false)
```

Comment:
volgens mij wel goed zo.

Ik zit alleen wat naar dat `sbg_code` te staren. Dat is wel heel erg algemeen. Wat als we straks iets anders ook via een code willen ondervangen? Wellicht is dan iets als `sbg_domcode` beter.